src/main/webapp/jsp/studentComments.jsp

```diff
@@ -1,297 +1,49 @@
-                                                            on <%=TimeHelper.formatDate(frc.createdAt)%> <%=frc.getEditedAtTextForStudent(frCommentGiver.equals("Anonymous"), lastEditorDisplay) %>
-                                                        </span>
-                                                    </div> <!-- frComment Content -->
-                                                    <div
-                                                        id="plainCommentText-<%=fsIndx%>-<%=qnIndx%>-<%=responseIndex%>-<%=responseCommentIndex%>"><%=frc.commentText.getValue()%></div>
-                                                </li>
-                                                <%
-                                                    }//FeedbackResponseComments loop ends
-                                                %>
-                                            </ul>
-                                        </td>
-                                    </tr>
-                                    <%
-                                        }//FeedbackResponse loop ends
-                                    %>
-                                </tbody>
-                            </table>
-                        </div>
-                        <%
-                            }//FeedbackQuestion loop ends
-                        %>
+                        You don't have any comment in this course.
                     </div>
                 </div>
-                <%
-                    }//FeedbackSession loop ends
-                %>
             </div>
-            <ul class="pagination">
-                <li><a href="<%=data.previousPageLink%>">«</a></li>
-                <%
-                    for (String courseId : data.coursePaginationList) {
-                %>
-                <li
-                    class="<%=courseId.equals(data.courseId) ? "active" : ""%>">
-                    <a
-                    href="<%=data.getStudentCommentsLink(false) + "&courseid=" + courseId%>"><%=courseId%></a>
-                </li>
-                <%
-                    }
-                %>
-                <li><a href="<%=data.nextPageLink%>">»</a></li>
-            </ul>
-            <% } else { %>
-            <div id="statusMessage" class="alert alert-warning">
-                There is no comment to display
-            </div>
-                
-            <% } %>
-    </div>
-    <jsp:include page="<%=Const.ViewURIs.FOOTER%>" />
-</body>
-</html>
\ No newline at end of file
+            <c:if test="${not empty data.commentRows}">
+                <br>
+                <comments:commentsForStudentsPanel commentRows="${data.commentRows}"/>
+            </c:if>
+            <c:forEach items="${data.feedbackSessionRows}" var="feedbackSessionRow" varStatus="h">
+                <br>
+                <comments:feedbackSessionPanel feedbackSessionRow="${feedbackSessionRow}" fsIdx="${h.index + 1}"/>
```

Comment:
It is the highest loop of a 4-level loop, so I thought I would use `h` so that the lowest level loop would use a `k` rather than an `l` :hushed:. I will probably change it to use `i` for all loops instead, since each loop is in their own separate tag/jsp file